examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+        "name": "edges_dp"
+    },
+    "input": {
+        "pfs": {
+            "glob": "/*",
+            "repo": "images_dp_1"
+        }
+    },
+    "output_branch": "dev",
+    "transform": {
+        "cmd": [ "python3", "/edges.py" ],
+        "image": "pachyderm/opencv"
+    }
+}
+```
+
+This means that this pipeline, 
+instead of putting its output in the `master` branch,
+will put it in a branch called `dev`.
+
+Since `montage_dp` is subscribed to the master branch of `edges_dp`,
+we can simply create a `master` branch attached to any commit in `edges_dp` to trigger the pipeline.
+
+## Example run-through
+
+### Deferred Processing
+
+You should have a Pachyderm cluster set up 
+and access to it configured from your local computer
+prior to running this example.
+
+1. Run the script `setup.sh` included in this repo.
+   It will perform the following steps for you:
+   
+   ```sh
+   pachctl create repo images_dp_1
+   pachctl create repo images_dp_2
+   pachctl create pipeline -f ./edges_dp.json 
+   pachctl create pipeline -f ./montage_dp.json 
+   pachctl put file images_dp_1@master -i ./images.txt
+   pachctl put file images_dp_1@master -i ./images2.txt
+   pachctl put file images_dp_2@master -i ./images3.txt
+   pachctl create branch edges_dp@stable_1_0 --head dev^
+   ```
+   
+2. Once the demo is loaded, 
+   look at the commits in `edges_dp`.
+   They should look something like this.
+   
+   ```sh
+   $  pachctl list commit edges_dp
+   REPO         BRANCH COMMIT                           FINISHED      SIZE PROGRESS DESCRIPTION
+   edges_dp     dev    85ca689439b943978d273b311003d1bf 2 minutes ago 0B   -         
+   edges_dp     dev    dc23aa4da6b94fefbebb6751c1c9b5a3 2 minutes ago 0B   -         
+   ```
+   
+3. Remember that the `edges_dp` pipeline outputs to the `dev` branch.
+   Since the `montage_dp` pipeline subscribes to the master branch,
+   it won't be triggered when `edges_dp` jobs complete,
+   since that output goes into the `dev` branch.
+
+4. See that there are two commits in `edges_dp`,
+   both in the `dev` branch that's the output for the pipeline.
+
+   ```sh
+   $ pachctl list commit edges_dp
+   REPO         BRANCH COMMIT                           FINISHED      SIZE PROGRESS DESCRIPTION
+   edges_dp     dev    85ca689439b943978d273b311003d1bf 2 minutes ago 0B   -         
+   edges_dp     dev    dc23aa4da6b94fefbebb6751c1c9b5a3 2 minutes ago 0B   -         
+   ```
+    
+5. We've also created a stable branch that points at that first commit,
+   with the `dev` branch at the most recent commit.
+   The `master` branch, of course, has no commits.
+   Take note of the commit ids and match them to the ids above.
+   
+   ```sh
+   $ pachctl list branch edges_dp
+   BRANCH     HEAD                             
+   stable_1_0 dc23aa4da6b94fefbebb6751c1c9b5a3 
+   master     -                                
+   dev        85ca689439b943978d273b311003d1bf 
+   ```
+
+6. If we look at the jobs for this DAG, 
```

Comment:
Again, this is an explanatory text. Keep it under step 5 with the indentation, but start your step 6 with an action, such as: 6. View the list of jobs: